### PR TITLE
bugfix: micronaut starter filter test

### PR DESCRIPTION
**Pull request by vitorbassozup**
### Related Issues

#1241 

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/android/beagle/src/main/java/br/com/zup/beagle/android/utils/ToolbarManager.kt#L196">android/beagle/src/main/java/br/com/zup/beagle/android/utils/ToolbarManager.kt#L196</a> - Prefer "<code>SHOW_AS_ACTION_IF_ROOM</code>" instead of "<code>SHOW_AS_ACTION_ALWAYS</code>"</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          6 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/beagle/example/pubspec.yaml#L15">beagle/example/pubspec.yaml#L15</a> - Sort pub dependencies</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/beagle/lib/action/beagle_alert.dart#L44">beagle/lib/action/beagle_alert.dart#L44</a> - 'FlatButton' is deprecated and shouldn't be used. Use TextButton instead. See the migration guide in flutter.dev/go/material-button-migration-guide). This feature was deprecated after v1.25.0-8.1.pre.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/beagle/pubspec.yaml#L21">beagle/pubspec.yaml#L21</a> - Sort pub dependencies</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/beagle_components/pubspec.yaml#L15">beagle_components/pubspec.yaml#L15</a> - Sort pub dependencies</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/beagle_components/pubspec.yaml#L23">beagle_components/pubspec.yaml#L23</a> - Sort pub dependencies</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/ac90c36f2249ab580dcda89c5045999fa0153283/sample/pubspec.yaml#L27">sample/pubspec.yaml#L27</a> - Sort pub dependencies</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_flutter">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/main/java/br/com/zup/beagle/android/components/LazyComponent.kt [hunk 1]
```diff
@@ -48,6 +55,27 @@ data class LazyComponent(
...
+                }
+
+
+            }
+        }
+    }
+
+    private fun addErrorScreen(beagleView: BeagleView, serverState: ServerDrivenState.Error) {
+        beagleView.removeAllViews()
+        val view = LayoutInflater.from(beagleView.context).inflate(R.layout.beagle_include_error_server_driven, null)
```
Comment (botbeaglezup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/components/LazyComponent.kt:73]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/4874db26ddf69cbe8511ab933be0862d62e8b091/android/beagle/src/main/java/br/com/zup/beagle/android/components/LazyComponent.kt#L73"></span>Avoid passing <code>null</code> as the view root (needed to resolve layout parameters on the inflated layout's root element)</td>
    </tr>
  </tbody>
</table>

**Comment by dantes-git** [comment]
This PR doesn't seem right anymore because it is listing a lot of commits that are already on the master

**Pull request closed without merging**